### PR TITLE
CASMINST-5093: Linting

### PR DESCRIPTION
**Pull request by mharding-hpe**
# Description

Linting (and linting-adjacent).

# Checklist Before Merging

- [X] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [X] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by mharding-hpe** [comment]
This should not merge until https://github.com/Cray-HPE/docs-csm/pull/2108 merges

**Comment by mharding-hpe** [comment]
Working on the backports for this, but this can be reviewed in the meantime

File: operations/conman/Troubleshoot_ConMan_Failing_to_Connect_to_a_Console.md [hunk 1]
```diff
@@ -62,19 +58,24 @@ This procedure requires administrative privileges.
         ncn-w003
         ```
 
-    1. `ssh` to the worker node that the pod is running on.
+    1. (`ncn#`) SSH to the worker node that the pod is running on.
```
Comment (mharding-hpe) [inline operations/conman/Troubleshoot_ConMan_Failing_to_Connect_to_a_Console.md:61]:
```suggestion
    1. (`ncn-mw#`) SSH to the worker node that the pod is running on.
```

File: operations/conman/Troubleshoot_ConMan_Failing_to_Connect_to_a_Console.md [hunk 2]
```diff
@@ -83,21 +84,21 @@ This procedure requires administrative privileges.
         ```
 
         This indicates there is a network issue between the worker node and the node of
```
Comment (mharding-hpe) [inline operations/conman/Troubleshoot_ConMan_Failing_to_Connect_to_a_Console.md:86]:
```suggestion
        This indicates that there is a network issue between the worker node and the node of
```

File: operations/hardware_state_manager/Set_BMC_Management_Role.md [hunk 3]
```diff
@@ -8,21 +8,30 @@ the Hardware State Manager (HSM), just like their associated nodes.
...
+  * See [Ignore Node within FAS](../firmware/FAS_Admin_Procedures.md#ignore).
+* Cray Advanced Platform Monitoring and Control (CAPMC)
+  * See [Ignore Nodes with CAPMC](../power_management/Ignore_Nodes_with_CAPMC.md).
 
 ## Topics
 
-* [When To Set BMC Management Role](#when-to-set-management-role-on-ncn-bmcs)
-* [How To Set BMC Management Role](#how-to-set-management-role-on-ncn-bmcs)
+* [Prerequisites](#prerequisites)
+* [When to set BMC management role](#when-to-set-management-role-on-ncn-bmcs)
```
Comment (mharding-hpe) [inline operations/hardware_state_manager/Set_BMC_Management_Role.md:21]:
```suggestion
* [When to set BMC management role](#when-to-set-bmc-management-role)
```

File: operations/hardware_state_manager/Set_BMC_Management_Role.md [hunk 4]
```diff
@@ -8,21 +8,30 @@ the Hardware State Manager (HSM), just like their associated nodes.
...
+* [How to set BMC management role](#how-to-set-management-role-on-ncn-bmcs)
```
Comment (mharding-hpe) [inline operations/hardware_state_manager/Set_BMC_Management_Role.md:22]:
```suggestion
* [How to set BMC management role](#how-to-set-bmc-management-role)
```

File: operations/node_management/Add_Remove_Replace_NCNs.md [hunk 5]
```diff
@@ -11,23 +11,24 @@ Add, remove, replace, or move non-compute nodes (NCNs). This applies to worker,
...
+  * [Replace NCN procedure](#replace-ncn-procedure)
 
 ## Prerequisites
 
 The system is fully installed and has transitioned off of the LiveCD.
 
 All activities required for site maintenance are complete.
 
-The latest `docs-csm` RPM has been installed on the master nodes. See [Check for Latest Documentation](../../update_product_stream/README.md#check-for-latest-documentation)
+The latest CSM documentation has been installed on the master nodes. See [Check for Latest Documentation](../../update_product_stream/README.md#check-for-latest-documentation)
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs.md:29]:
```suggestion
The latest CSM documentation has been installed on the master nodes. See [Check for Latest Documentation](../../update_product_stream/README.md#check-for-latest-documentation).
```

File: operations/node_management/Add_Remove_Replace_NCNs.md [hunk 6]
```diff
@@ -229,121 +237,108 @@ echo $XNAME
...
-      {
-         "error": {
-            "code": "iLO.0.10.ExtendedInfo",
-            "message": "See @Message.ExtendedInfo for more information.",
-            "@Message.ExtendedInfo": [
-               {
-               "MessageId": "iLO.2.14.ResetRequired"
-               }
-            ]
+   1. If IPMI is disabled, then enable IPMI and restart the BMC.
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs.md:240]:
```suggestion
   1. (`ncn-m#`) If IPMI is disabled, then enable IPMI and restart the BMC.
```

File: operations/node_management/Add_Remove_Replace_NCNs.md [hunk 7]
```diff
@@ -229,121 +237,108 @@ echo $XNAME
...
+
+(`ncn#`) For several of the commands in this section, variables must be set with the name of the node being removed and its component name (xname).
 Set `NODE` to the hostname of the node being removed (for example `ncn-w001`, `ncn-s002`, etc).
 Set `XNAME` to the xname of that node.
 
 ```bash
 NODE=ncn-x00n
-XNAME=$(ssh $NODE cat /etc/cray/xname)
-echo $XNAME
+XNAME=$(ssh ${NODE} cat /etc/cray/xname)
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs.md:317]:
```suggestion
XNAME=$(ssh ${NODE} cat /etc/cray/xname)
echo "${XNAME}"
```

File: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md [hunk 8]
```diff
@@ -145,18 +148,18 @@ Scenarios where this procedure is applicable:
...
 
             ```bash
-            export BMC_IP=$(curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST -H "Content-Type: application/json" \
+            BMC_IP=$(curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST -H "Content-Type: application/json" \
                                     -d '{ "command": "lease4-get-all", "service": [ "dhcp4" ] }' \
                                     https://api-gw-service-nmn.local/apis/dhcp-kea |
-                                jq --arg BMC_MAC "$BMC_MAC" \
+                                jq --arg BMC_MAC "${BMC_MAC}" \
                                     '.[].arguments.leases[] | select(."hw-address" == $BMC_MAC)."ip-address"' -r)
             echo $BMC_IP
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:164]:
```suggestion
            echo ${BMC_IP}
```

File: operations/node_management/Add_Remove_Replace_NCNs/Boot_NCN.md [hunk 9]
```diff
@@ -6,64 +6,65 @@ Boot a master, worker, or storage non-compute node (NCN) that is to be added to
...
    ***Open this link in a new tab or page:*** [Log in to a Node Using ConMan](../../conman/Log_in_to_a_Node_Using_ConMan.md)
 
    The first session will be needed to run the commands in the following Rebuild Node steps.
 
-### Set the PXE Boot Option and Power on the Node
+### Set the PXE boot option and power on the node
 
 **IMPORTANT:** These commands assume that the variables from [the prerequisites section](../Add_Remove_Replace_NCNs.md#add-ncn-prerequisites) have been set.
 
 1. Set the `BMC` variable to the hostname of the BMC of the node being rebuilt. If booting `ncn-m001`, set this to the FQDN or IP address.
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Boot_NCN.md:21]:
```suggestion
1. Set the `BMC` variable to the hostname of the BMC of the node being rebuilt.

    If booting `ncn-m001`, then set this to the FQDN or IP address.
```

File: operations/node_management/Add_Remove_Replace_NCNs/Collect_NCN_MAC_Addresses.md [hunk 10]
```diff
@@ -88,7 +97,9 @@ A temporary MAC address collection iPXE bootscript is put into place on the syst
         ipmitool -I lanplus -U root -E -H $BMC_IP chassis power on
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Collect_NCN_MAC_Addresses.md:97]:
```suggestion
        ipmitool -I lanplus -U root -E -H "${BMC_IP}" chassis power on
```

File: operations/node_management/Add_Remove_Replace_NCNs/Collect_NCN_MAC_Addresses.md [hunk 11]
```diff
@@ -115,40 +126,41 @@ A temporary MAC address collection iPXE bootscript is put into place on the syst
         ipmitool -I lanplus -U root -E -H $BMC_IP chassis power off
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Collect_NCN_MAC_Addresses.md:126]:
```suggestion
        ipmitool -I lanplus -U root -E -H "${BMC_IP}" chassis power off
```

File: operations/node_management/Add_Remove_Replace_NCNs/Collect_NCN_MAC_Addresses.md [hunk 12]
```diff
@@ -112,43 +123,44 @@ A temporary MAC address collection iPXE bootscript is put into place on the syst
...
 
         ```bash
         sleep 30
         kubectl -n services logs -l app.kubernetes.io/name=cray-ipxe -c cray-ipxe -f
         ```
 
-        The following output means the new iPXE binary has been built.
+        The following output means that a new iPXE binary has been built.
 
         ```bash
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Collect_NCN_MAC_Addresses.md:161]:
```suggestion
        ```text
```

File: operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_from_Role.md [hunk 13]
```diff
@@ -139,23 +141,25 @@ Run the commands in this section **on the node being removed**. The commands can
...
-#### Remove the Master Node from the Kubernetes Cluster
+#### Remove the master node from the Kubernetes cluster
 
-**IMPORTANT:** Run this command from a node that is ***NOT*** being deleted.
+**IMPORTANT:** Run this command from a master or worker node that is ***NOT*** being deleted.
+
+(`ncn-mw#`) Remove the master node from the Kubernetes cluster.
 
 ```bash
 kubectl delete node $NODE
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_from_Role.md:157]:
```suggestion
kubectl delete node "${NODE}"
```

**Review by rustydb — APPROVED** [review]

File: operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md [hunk 14]
```diff
@@ -30,81 +30,96 @@ This section applies to all node types. The commands in this section assume the
...
+   **Troubleshooting:**
 
-       * **Troubleshooting:** If the `NBP file...` output never appears, or something else goes wrong, go back to the steps for modifying the `XNAME.json` file (see the step to [inspect and modify the JSON file](Identify_Nodes_and_Update_Metadata.md#Inspect-and-modify-the-JSON-file) and make sure these instructions were completed correctly.
+   * If the `NBP file...` output never appears, or something else goes wrong, go back to the steps for modifying the `XNAME.json` file (see the step to
+     [inspect and modify the JSON file](Identify_Nodes_and_Update_Metadata.md#Inspect-and-modify-the-JSON-file) and make sure these instructions were completed correctly.
 
-    * **Master nodes only:** If `cloud-init` did not complete the newly rebuilt node will need to have its `etcd` service definition manually updated. Reconfigure the `etcd` service, and restart `cloud-init` on the newly rebuilt master:
+   * **Master nodes only:** If `cloud-init` did not complete the newly rebuilt node will need to have its `etcd` service definition manually updated. Reconfigure the `etcd` service, and
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md:88]:
```suggestion
   * **Master nodes only:** If `cloud-init` did not complete, then the newly rebuilt node will need to have its `etcd` service definition manually updated. Reconfigure the `etcd` service and
```

File: operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md [hunk 15]
```diff
@@ -30,81 +30,96 @@ This section applies to all node types. The commands in this section assume the
...
+   ```text
+   [  295.466827] cloud-init[9333]: Cloud-init v. 20.2-8.45.1 running 'modules:final' at Thu, 26 Aug2021  15:23:20 +0000. Up 125.72 seconds.
+   [  295.467037] cloud-init[9333]: Cloud-init v. 20.2-8.45.1 finished at Thu, 26 Aug 2021 15:26:12+0000. Datasource DataSourceNoCloudNet [seed=cmdline,http://10.92.100.81:8888/][dsmode=net].  Up 29546 seconds
+   ```
 
-   1. Press enter on the console to ensure that the the login prompt is displayed including the correct hostname of this node. Then exit the ConMan console (`&` then `.`), and then use `ssh` to log in to the node to complete the remaining validation steps.
+   **Troubleshooting:**
 
-       * **Troubleshooting:** If the `NBP file...` output never appears, or something else goes wrong, go back to the steps for modifying the `XNAME.json` file (see the step to [inspect and modify the JSON file](Identify_Nodes_and_Update_Metadata.md#Inspect-and-modify-the-JSON-file) and make sure these instructions were completed correctly.
+   * If the `NBP file...` output never appears, or something else goes wrong, go back to the steps for modifying the `XNAME.json` file (see the step to
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md:85]:
```suggestion
   * If the `NBP file...` output never appears, or something else goes wrong, then go back to the steps for modifying the `XNAME.json` file (see the step to
```

File: operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md [hunk 16]
```diff
@@ -130,25 +144,25 @@ This section applies to all node types. The commands in this section assume the
        curl -i -s -k -H "Content-Type: application/json" \
            -H "Authorization: Bearer ${TOKEN}" \
            "https://api-gw-service-nmn.local/apis/bss/boot/v1/bootparameters" \
-           -X PUT -d @./${XNAME}.json
+           -X PUT -d @"./${XNAME}.json"
        ```
 
    1. Verify the `bss bootparameters list` command returns the expected information.
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md:150]:
```suggestion
   1. Verify that the `bss bootparameters list` command returns the expected information.
```

File: operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md [hunk 17]
```diff
@@ -106,15 +123,16 @@ An authentication token is required to access the API gateway and to use the `sa
    Aborting.
    ```
 
-   In the preceding example, the commands to stop containers timed out on all the worker nodes and reported `WARNING` and `ERROR` messages. A summary of the issue displays and prompts the user to continue or stop. Respond `no` stop the shutdown. Then review the containers running on the nodes.
+   In the preceding example, the commands to stop containers timed out on all the worker nodes and reported `WARNING` and `ERROR` messages.
+   A summary of the issue displays and prompts the user to continue or stop. Respond `no` stop the shutdown. Then review the containers running on the nodes.
 
    ```bash
    for ncn in ncn-w00{1,2,3}; do echo "$ncn"; ssh $ncn "crictl ps"; echo; done
```
Comment (mharding-hpe) [inline operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md:130]:
```suggestion
   for ncn in ncn-w00{1,2,3}; do echo "${ncn}"; ssh "${ncn}" "crictl ps"; echo; done
```

File: operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md [hunk 18]
```diff
@@ -225,54 +246,54 @@ An authentication token is required to access the API gateway and to use the `sa
...
 1. Wait until the console indicates that the node has shut down.
 
-1. From a remote system that has access to the management plane, use IPMI tool to power off `ncn-m001`.
+1. (`external#`) From a remote system that has access to the management plane, power off `ncn-m001`.
 
     ```bash
-    remote$ ipmitool -I lanplus -U $USERNAME -E -H NCN-M001_BMC_HOSTNAME chassis power status
-    remote$ ipmitool -I lanplus -U $USERNAME -E -H NCN-M001_BMC_HOSTNAME chassis power off
-    remote$ ipmitool -I lanplus -U $USERNAME -E -H NCN-M001_BMC_HOSTNAME chassis power status
+    ipmitool -I lanplus -U $USERNAME -E -H NCN-M001_BMC_HOSTNAME chassis power status
```
Comment (mharding-hpe) [inline operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md:285]:
```suggestion
    ipmitool -I lanplus -U "${USERNAME}" -E -H NCN-M001_BMC_HOSTNAME chassis power status
```

File: operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md [hunk 19]
```diff
@@ -225,54 +246,54 @@ An authentication token is required to access the API gateway and to use the `sa
...
+    ipmitool -I lanplus -U $USERNAME -E -H NCN-M001_BMC_HOSTNAME chassis power off
```
Comment (mharding-hpe) [inline operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md:286]:
```suggestion
    ipmitool -I lanplus -U "${USERNAME}" -E -H NCN-M001_BMC_HOSTNAME chassis power off
```

File: operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md [hunk 20]
```diff
@@ -225,54 +246,54 @@ An authentication token is required to access the API gateway and to use the `sa
...
+    ipmitool -I lanplus -U $USERNAME -E -H NCN-M001_BMC_HOSTNAME chassis power status
```
Comment (mharding-hpe) [inline operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md:287]:
```suggestion
    ipmitool -I lanplus -U "${USERNAME}" -E -H NCN-M001_BMC_HOSTNAME chassis power status
```

**Review by nrockershousen — APPROVED** [review]

File: operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md [hunk 21]
```diff
@@ -21,7 +21,7 @@ This section applies to all node types. The commands in this section assume the
     1. Set the BMC variable to the hostname of the BMC of the node being rebuilt.
 
         ```bash
-        BMC=${NODE}-mgmt
+        BMC="${NODE}-mgmt"
         ```
 
     1. Set and export the root password of the BMC.
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md:27]:
```suggestion
    1. Set and export the `root` password of the BMC.
```

File: operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md [hunk 22]
```diff
@@ -21,90 +21,105 @@ This section applies to all node types. The commands in this section assume the
...
+1. (`ncn#`) Set the wipe flag back so it will not wipe the disk when the node is rebooted.
 
-1. Set the wipe flag back so it will not wipe the disk when the node is rebooted.
+   1. Run the following commands from a node that has `cray` CLI initialized.
 
-   1. Run the following commands from a node that has `cray` CLI initialized:
+      See [Configure the Cray CLI](../../configure_cray_cli.md).
 
        ```bash
        cray bss bootparameters list --name $XNAME --format=json | jq .[] > ${XNAME}.json
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md:125]:
```suggestion
       cray bss bootparameters list --name "${XNAME}" --format=json | jq .[] > "${XNAME}.json"
```

**Pull request merged**